README.md

```diff
@@ -36,7 +36,7 @@ deploy-code:
 
     steps:
```

Comment:
Like the comment above I don't have a good place to put this but we need to update the Description for the `github-token` parameter on the Readme.  It needs to indicate that if you are downloading an asset from another repository you need to provide something besides ${{ secrets.GITHUB_TOKEN }} because that token does not have access to other repositories.  It may seem obvious but we want to put as much info in there as possible to avoid questions.

Reply:
You wouldn't happen to have a link to your git docs that speaks to creating a github specific token for this scenario?  Something I could include in the description?